client/src/components/annotators/useMediaController.ts

```diff
@@ -319,14 +297,14 @@ export function useMediaController() {
           imageCursorRef.value.style.display = 'block';
         }
       },
-      handleMouseMove(evt: MouseEvent) {
-        const offsetX = evt.clientX + 10;
-        const offsetY = evt.clientY - 25;
-        window.requestAnimationFrame(() => {
-          if (imageCursorRef.value) {
-            imageCursorRef.value.style.transform = `translate(${offsetX}px, ${offsetY}px)`;
-          }
-        });
+      handleMouseMove(/*evt: MouseEvent*/) {
+        // const offsetX = evt.clientX + 10;
+        // const offsetY = evt.clientY - 25;
+        // window.requestAnimationFrame(() => {
+        //   if (imageCursorRef.value) {
+        //     imageCursorRef.value.style.transform = `translate(${offsetX}px, ${offsetY}px)`;
+        //   }
+        // });
```

Comment:
is this intended to be handled outside of the individual cameras?
There is some stuff that I probably need to do to get mouse-trap shortcuts working as well as proper behavior of being in a 'new' state and selecting a different camera before completing an annotation.

Reply:
I couldn't figure out what this did, so I disabled it to see if anything broke.

It doesn't appear to be doing anything.  What is this for?

Reply:
This handles the icon for new items attached to the mouse.
![image](https://user-images.githubusercontent.com/61746913/154548049-70161e2c-c13f-4e71-8b8e-7c4254768cb7.png)

Only in creation mode does this icon come up.

Reply:
OK, thanks.  Not sure why I didn't notice it.

Reply:
I'll fix this now, and try and resolve the CI failure.